indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -292,6 +293,20 @@ public Set<ResourceAction> getInputSourceResources()
            ImmutableSet.of();
   }
 
+  @Nullable
```

Comment:
imo it makes more sense to have skipPublishingReports be on the ParallelIndexSupervisorTask rather than the ParallelIndexIngestionSpec. the spec is user facing and i don't think we would ever want this field to be user settable

Reply:
Done.